{
  "build": "py38h06a4308_0",
  "build_number": 0,
  "channel": "https://repo.anaconda.com/pkgs/main/linux-64",
  "constrains": [],
  "depends": [
    "jsonschema >=4.18.0",
    "python >=3.8,<3.9.0a0",
    "python-json-logger >=2.0.4",
    "pyyaml >=5.3",
    "referencing",
    "rfc3339-validator",
    "rfc3986-validator >=0.1.1",
    "traitlets >=5.3"
  ],
  "extracted_package_dir": "/opt/conda/pkgs/jupyter_events-0.10.0-py38h06a4308_0",
  "files": [
    "bin/jupyter-events",
    "lib/python3.8/site-packages/jupyter_events-0.10.0.dist-info/INSTALLER",
    "lib/python3.8/site-packages/jupyter_events-0.10.0.dist-info/METADATA",
    "lib/python3.8/site-packages/jupyter_events-0.10.0.dist-info/RECORD",
    "lib/python3.8/site-packages/jupyter_events-0.10.0.dist-info/REQUESTED",
    "lib/python3.8/site-packages/jupyter_events-0.10.0.dist-info/WHEEL",
    "lib/python3.8/site-packages/jupyter_events-0.10.0.dist-info/direct_url.json",
    "lib/python3.8/site-packages/jupyter_events-0.10.0.dist-info/entry_points.txt",
    "lib/python3.8/site-packages/jupyter_events-0.10.0.dist-info/licenses/LICENSE",
    "lib/python3.8/site-packages/jupyter_events/__init__.py",
    "lib/python3.8/site-packages/jupyter_events/__pycache__/__init__.cpython-38.pyc",
    "lib/python3.8/site-packages/jupyter_events/__pycache__/_version.cpython-38.pyc",
    "lib/python3.8/site-packages/jupyter_events/__pycache__/cli.cpython-38.pyc",
    "lib/python3.8/site-packages/jupyter_events/__pycache__/logger.cpython-38.pyc",
    "lib/python3.8/site-packages/jupyter_events/__pycache__/pytest_plugin.cpython-38.pyc",
    "lib/python3.8/site-packages/jupyter_events/__pycache__/schema.cpython-38.pyc",
    "lib/python3.8/site-packages/jupyter_events/__pycache__/schema_registry.cpython-38.pyc",
    "lib/python3.8/site-packages/jupyter_events/__pycache__/traits.cpython-38.pyc",
    "lib/python3.8/site-packages/jupyter_events/__pycache__/validators.cpython-38.pyc",
    "lib/python3.8/site-packages/jupyter_events/__pycache__/yaml.cpython-38.pyc",
    "lib/python3.8/site-packages/jupyter_events/_version.py",
    "lib/python3.8/site-packages/jupyter_events/cli.py",
    "lib/python3.8/site-packages/jupyter_events/logger.py",
    "lib/python3.8/site-packages/jupyter_events/py.typed",
    "lib/python3.8/site-packages/jupyter_events/pytest_plugin.py",
    "lib/python3.8/site-packages/jupyter_events/schema.py",
    "lib/python3.8/site-packages/jupyter_events/schema_registry.py",
    "lib/python3.8/site-packages/jupyter_events/schemas/event-core-schema.yml",
    "lib/python3.8/site-packages/jupyter_events/schemas/event-metaschema.yml",
    "lib/python3.8/site-packages/jupyter_events/schemas/property-metaschema.yml",
    "lib/python3.8/site-packages/jupyter_events/traits.py",
    "lib/python3.8/site-packages/jupyter_events/validators.py",
    "lib/python3.8/site-packages/jupyter_events/yaml.py"
  ],
  "fn": "jupyter_events-0.10.0-py38h06a4308_0.conda",
  "license": "BSD-3-Clause",
  "link": {
    "source": "/opt/conda/pkgs/jupyter_events-0.10.0-py38h06a4308_0",
    "type": 1
  },
  "md5": "45d66de58c846caf29d69b1d865b106f",
  "name": "jupyter_events",
  "package_tarball_full_path": "/opt/conda/pkgs/jupyter_events-0.10.0-py38h06a4308_0.conda",
  "paths_data": {
    "paths": [
      {
        "_path": "bin/jupyter-events",
        "file_mode": "text",
        "path_type": "hardlink",
        "prefix_placeholder": "/croot/jupyter_events_1718738097486/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl",
        "sha256": "276d6713e933d8ad8ccec4f03020d630e0a614fcaab811d6e39d891ca73e96e3",
        "sha256_in_prefix": "fbe36d64a3231ff9becf145a8a2fbe0a97d93718e2a8ce2d5cc2f6c981d18740",
        "size_in_bytes": 471
      },
      {
        "_path": "lib/python3.8/site-packages/jupyter_events-0.10.0.dist-info/INSTALLER",
        "path_type": "hardlink",
        "sha256": "d0edee15f91b406f3f99726e44eb990be6e34fd0345b52b910c568e0eef6a2a8",
        "sha256_in_prefix": "d0edee15f91b406f3f99726e44eb990be6e34fd0345b52b910c568e0eef6a2a8",
        "size_in_bytes": 5
      },
      {
        "_path": "lib/python3.8/site-packages/jupyter_events-0.10.0.dist-info/METADATA",
        "path_type": "hardlink",
        "sha256": "f8ca2d8a79b25b77398e52ef9d3cfae48388744b0ad3f88221dcb330ae9ace33",
        "sha256_in_prefix": "f8ca2d8a79b25b77398e52ef9d3cfae48388744b0ad3f88221dcb330ae9ace33",
        "size_in_bytes": 5714
      },
      {
        "_path": "lib/python3.8/site-packages/jupyter_events-0.10.0.dist-info/RECORD",
        "path_type": "hardlink",
        "sha256": "36da78d7574642bff3a4e15ddc7b58f3d1427fd5fd59380ea4c31a874bb38bc6",
        "sha256_in_prefix": "36da78d7574642bff3a4e15ddc7b58f3d1427fd5fd59380ea4c31a874bb38bc6",
        "size_in_bytes": 2581
      },
      {
        "_path": "lib/python3.8/site-packages/jupyter_events-0.10.0.dist-info/REQUESTED",
        "path_type": "hardlink",
        "sha256": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855",
        "sha256_in_prefix": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855",
        "size_in_bytes": 0
      },
      {
        "_path": "lib/python3.8/site-packages/jupyter_events-0.10.0.dist-info/WHEEL",
        "path_type": "hardlink",
        "sha256": "4c93e729db6b4ae7bbc59fc0546929f585dcbc3ae86ec8c176cd5dbb7371e9d7",
        "sha256_in_prefix": "4c93e729db6b4ae7bbc59fc0546929f585dcbc3ae86ec8c176cd5dbb7371e9d7",
        "size_in_bytes": 87
      },
      {
        "_path": "lib/python3.8/site-packages/jupyter_events-0.10.0.dist-info/direct_url.json",
        "path_type": "hardlink",
        "sha256": "70b974e0172f691cfe3988e06a790601e2fb004c75b82477fe79dcd374488d66",
        "sha256_in_prefix": "70b974e0172f691cfe3988e06a790601e2fb004c75b82477fe79dcd374488d66",
        "size_in_bytes": 74
      },
      {
        "_path": "lib/python3.8/site-packages/jupyter_events-0.10.0.dist-info/entry_points.txt",
        "path_type": "hardlink",
        "sha256": "da6e32437af24d8a74f2e98251e5c1a90d4b2a2c37b696c93ecb93ccf9f288de",
        "sha256_in_prefix": "da6e32437af24d8a74f2e98251e5c1a90d4b2a2c37b696c93ecb93ccf9f288de",
        "size_in_bytes": 59
      },
      {
        "_path": "lib/python3.8/site-packages/jupyter_events-0.10.0.dist-info/licenses/LICENSE",
        "path_type": "hardlink",
        "sha256": "80405b5460d62fb7032d62e4720fa199a5be01864c900f1b9de79bc811efd836",
        "sha256_in_prefix": "80405b5460d62fb7032d62e4720fa199a5be01864c900f1b9de79bc811efd836",
        "size_in_bytes": 1534
      },
      {
        "_path": "lib/python3.8/site-packages/jupyter_events/__init__.py",
        "path_type": "hardlink",
        "sha256": "ef1fb7618c7b8626bd723fdf7f1d52e69c4f62d108fdd14ac1bd19e43d9e2b0f",
        "sha256_in_prefix": "ef1fb7618c7b8626bd723fdf7f1d52e69c4f62d108fdd14ac1bd19e43d9e2b0f",
        "size_in_bytes": 222
      },
      {
        "_path": "lib/python3.8/site-packages/jupyter_events/__pycache__/__init__.cpython-38.pyc",
        "path_type": "hardlink",
        "sha256": "81f80baea791b7dd6edcc805c42a8ff24a55d8593c9b65fabf9dea0e034ee672",
        "sha256_in_prefix": "81f80baea791b7dd6edcc805c42a8ff24a55d8593c9b65fabf9dea0e034ee672",
        "size_in_bytes": 350
      },
      {
        "_path": "lib/python3.8/site-packages/jupyter_events/__pycache__/_version.cpython-38.pyc",
        "path_type": "hardlink",
        "sha256": "ab03b68fea935c8170f050fb7b8b3dafc5aa29e943f741c76a10322818ff7c8f",
        "sha256_in_prefix": "ab03b68fea935c8170f050fb7b8b3dafc5aa29e943f741c76a10322818ff7c8f",
        "size_in_bytes": 815
      },
      {
        "_path": "lib/python3.8/site-packages/jupyter_events/__pycache__/cli.cpython-38.pyc",
        "path_type": "hardlink",
        "sha256": "c69d1827419d306b4300cf6ef096f23b5d3dee78a91d7c2654d9ff2f651881ec",
        "sha256_in_prefix": "c69d1827419d306b4300cf6ef096f23b5d3dee78a91d7c2654d9ff2f651881ec",
        "size_in_bytes": 3005
      },
      {
        "_path": "lib/python3.8/site-packages/jupyter_events/__pycache__/logger.cpython-38.pyc",
        "path_type": "hardlink",
        "sha256": "f3d6cfebe2767c490a7eb30d141a8e7fc894f9dcac2c0387c161b6b5a0576d38",
        "sha256_in_prefix": "f3d6cfebe2767c490a7eb30d141a8e7fc894f9dcac2c0387c161b6b5a0576d38",
        "size_in_bytes": 11663
      },
      {
        "_path": "lib/python3.8/site-packages/jupyter_events/__pycache__/pytest_plugin.cpython-38.pyc",
        "path_type": "hardlink",
        "sha256": "a44bac18f8185ffd9e1049a476f2f4d6ceecbe83fb6f05ab509eafbf380d36b6",
        "sha256_in_prefix": "a44bac18f8185ffd9e1049a476f2f4d6ceecbe83fb6f05ab509eafbf380d36b6",
        "size_in_bytes": 2321
      },
      {
        "_path": "lib/python3.8/site-packages/jupyter_events/__pycache__/schema.cpython-38.pyc",
        "path_type": "hardlink",
        "sha256": "dbf6a3f675ad2c9386f9a029a7e091dff9ee977434ef2f42a866f6e563ddebd5",
        "sha256_in_prefix": "dbf6a3f675ad2c9386f9a029a7e091dff9ee977434ef2f42a866f6e563ddebd5",
        "size_in_bytes": 5256
      },
      {
        "_path": "lib/python3.8/site-packages/jupyter_events/__pycache__/schema_registry.cpython-38.pyc",
        "path_type": "hardlink",
        "sha256": "8de478b21e1fcd225ff9289748d06c25f386d2452ff02735672f6ff3b7e0af72",
        "sha256_in_prefix": "8de478b21e1fcd225ff9289748d06c25f386d2452ff02735672f6ff3b7e0af72",
        "size_in_bytes": 3525
      },
      {
        "_path": "lib/python3.8/site-packages/jupyter_events/__pycache__/traits.cpython-38.pyc",
        "path_type": "hardlink",
        "sha256": "e0483d66222fd86863b0d3bcf9aad56db4cc3bf1563d5974d9f6a7dbb6eb798d",
        "sha256_in_prefix": "e0483d66222fd86863b0d3bcf9aad56db4cc3bf1563d5974d9f6a7dbb6eb798d",
        "size_in_bytes": 1705
      },
      {
        "_path": "lib/python3.8/site-packages/jupyter_events/__pycache__/validators.cpython-38.pyc",
        "path_type": "hardlink",
        "sha256": "6d790dcd390336cdf902286da43c11376a796ff4638cf43772c19b01907ba774",
        "sha256_in_prefix": "6d790dcd390336cdf902286da43c11376a796ff4638cf43772c19b01907ba774",
        "size_in_bytes": 2013
      },
      {
        "_path": "lib/python3.8/site-packages/jupyter_events/__pycache__/yaml.cpython-38.pyc",
        "path_type": "hardlink",
        "sha256": "ad609c49b3255e8b9ba9215c63f86abcbd8854acb379c29e5c1c686a10743bde",
        "sha256_in_prefix": "ad609c49b3255e8b9ba9215c63f86abcbd8854acb379c29e5c1c686a10743bde",
        "size_in_bytes": 1393
      },
      {
        "_path": "lib/python3.8/site-packages/jupyter_events/_version.py",
        "path_type": "hardlink",
        "sha256": "1379afb135572376d3f7da8960f7113b3bf7e26868b51613767ae0462e8cc755",
        "sha256_in_prefix": "1379afb135572376d3f7da8960f7113b3bf7e26868b51613767ae0462e8cc755",
        "size_in_bytes": 648
      },
      {
        "_path": "lib/python3.8/site-packages/jupyter_events/cli.py",
        "path_type": "hardlink",
        "sha256": "fc226693bd54009e135f5c3b5a247ffe150a205064cbc8f62b67aeab4658d177",
        "sha256_in_prefix": "fc226693bd54009e135f5c3b5a247ffe150a205064cbc8f62b67aeab4658d177",
        "size_in_bytes": 3087
      },
      {
        "_path": "lib/python3.8/site-packages/jupyter_events/logger.py",
        "path_type": "hardlink",
        "sha256": "3092bce56b33bdf76cb05990d4f35b1126d63e02ae51524a2e9c38906006777f",
        "sha256_in_prefix": "3092bce56b33bdf76cb05990d4f35b1126d63e02ae51524a2e9c38906006777f",
        "size_in_bytes": 15842
      },
      {
        "_path": "lib/python3.8/site-packages/jupyter_events/py.typed",
        "path_type": "hardlink",
        "sha256": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855",
        "sha256_in_prefix": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855",
        "size_in_bytes": 0
      },
      {
        "_path": "lib/python3.8/site-packages/jupyter_events/pytest_plugin.py",
        "path_type": "hardlink",
        "sha256": "1b4182cfb43fdf21ed02df73d00a2a64d57e6d06efe79defbe3d8c303cf2ede2",
        "sha256_in_prefix": "1b4182cfb43fdf21ed02df73d00a2a64d57e6d06efe79defbe3d8c303cf2ede2",
        "size_in_bytes": 1778
      },
      {
        "_path": "lib/python3.8/site-packages/jupyter_events/schema.py",
        "path_type": "hardlink",
        "sha256": "00bebc037267c3d40830ccde331b310d9833b906f2201e47b4cd13e9bb43e019",
        "sha256_in_prefix": "00bebc037267c3d40830ccde331b310d9833b906f2201e47b4cd13e9bb43e019",
        "size_in_bytes": 5390
      },
      {
        "_path": "lib/python3.8/site-packages/jupyter_events/schema_registry.py",
        "path_type": "hardlink",
        "sha256": "0a78b0a27490a3731cd2d44b2fe89576c9ada1a0788dfb8a9a67492bc6e3f78d",
        "sha256_in_prefix": "0a78b0a27490a3731cd2d44b2fe89576c9ada1a0788dfb8a9a67492bc6e3f78d",
        "size_in_bytes": 2762
      },
      {
        "_path": "lib/python3.8/site-packages/jupyter_events/schemas/event-core-schema.yml",
        "path_type": "hardlink",
        "sha256": "c4fc7e1b9d162386463dde84fdb6564293503c71ac306b29d808b0cbcc6baccb",
        "sha256_in_prefix": "c4fc7e1b9d162386463dde84fdb6564293503c71ac306b29d808b0cbcc6baccb",
        "size_in_bytes": 580
      },
      {
        "_path": "lib/python3.8/site-packages/jupyter_events/schemas/event-metaschema.yml",
        "path_type": "hardlink",
        "sha256": "721d42ebc8de49053263a4ba000a51f701670236f1dbe79cf9b380b2e8ac50eb",
        "sha256_in_prefix": "721d42ebc8de49053263a4ba000a51f701670236f1dbe79cf9b380b2e8ac50eb",
        "size_in_bytes": 626
      },
      {
        "_path": "lib/python3.8/site-packages/jupyter_events/schemas/property-metaschema.yml",
        "path_type": "hardlink",
        "sha256": "9a2ab66d1473a3d9bf7a154ad6f6316f0c78e91847e2e0e5212fa5a72bc686e3",
        "sha256_in_prefix": "9a2ab66d1473a3d9bf7a154ad6f6316f0c78e91847e2e0e5212fa5a72bc686e3",
        "size_in_bytes": 624
      },
      {
        "_path": "lib/python3.8/site-packages/jupyter_events/traits.py",
        "path_type": "hardlink",
        "sha256": "f78753814a2a4d36a8fc73cfbc701853069e2aaaf8154115dbb8d5ba7120a603",
        "sha256_in_prefix": "f78753814a2a4d36a8fc73cfbc701853069e2aaaf8154115dbb8d5ba7120a603",
        "size_in_bytes": 1818
      },
      {
        "_path": "lib/python3.8/site-packages/jupyter_events/validators.py",
        "path_type": "hardlink",
        "sha256": "94f27a8c350b51e9839c617b8e778de6ca3629391e3c8c5e08f3b7408d1403bb",
        "sha256_in_prefix": "94f27a8c350b51e9839c617b8e778de6ca3629391e3c8c5e08f3b7408d1403bb",
        "size_in_bytes": 2377
      },
      {
        "_path": "lib/python3.8/site-packages/jupyter_events/yaml.py",
        "path_type": "hardlink",
        "sha256": "13fb01b745636e1a45caed4b302806db3d59b57eba0b26a3f989130a37534469",
        "sha256_in_prefix": "13fb01b745636e1a45caed4b302806db3d59b57eba0b26a3f989130a37534469",
        "size_in_bytes": 1031
      }
    ],
    "paths_version": 1
  },
  "requested_spec": "None",
  "sha256": "9e98ee98480428d7caf85905a8b7a3d61bd9e56688121eb6220b68b0b80dd0c4",
  "size": 37435,
  "subdir": "linux-64",
  "timestamp": 1718738195000,
  "url": "https://repo.anaconda.com/pkgs/main/linux-64/jupyter_events-0.10.0-py38h06a4308_0.conda",
  "version": "0.10.0"
}